samples/gramine-hello/hello.manifest.template

```diff
@@ -1,4 +1,4 @@
-loader.preload = "file:{{ gramine.libos }}"
+loader.entrypoint = "file:{{ gramine.libos }}"
 loader.env.LD_LIBRARY_PATH = "/lib"
 
```

Comment:
have you checked whether the new format requires changes to the gramine-prepare command?